### ISSUE TITLE
Bar chart xaxis/yaxis confusion

### ISSUE THREAD

**Issue by JCQuintas** [apexcharts/apexcharts.js#87]
When you use the "bar" chart, the xaxis and yaxis are inverted. And that causes a bit of confusion doesn't it?

Could we have it so xaxis controls the real xaxis and yaxis controls the real yaxis?

**Comment by junedchhipa** [comment]
You cannot imagine the amount of confusion I had while building the horizontal bar charts. I had to turn my head 90 degrees everytime I had to deal with x-axis/y-axis of bar-charts ;)

But the thing is, it is actually the correct way to deal with it as I have researched other charts as well before developing the configurations for it.

See Highcharts horizontal bar chart: https://www.highcharts.com/demo/bar-stacked
as well as I saw a similar example for d3 too.

So, let's stick to the standards.

**Comment by junedchhipa** [comment]
Yes, indeed. I will keep an eye on how it can be done in better ways.

**Comment by srweal** [comment]
I like your library - great job.

I was going to report this bug myself, but saw this closed issue.

I think this issue should be reopened as I believe it is a non-standard implementation. The example you provide from Highcharts puts a title on the 'real y-axis', which then gets shown on the rotated bar chart (i.e. on the bottom axis). The current ApexChart version applies this incorrectly and results in x-axis settings being applied to the rotated y-axis. 

There are also some options that only apply logically to the x-axis data (e.g. date formatting, trimming of labels). Hopefully it is an easy bug fix.
